### ISSUE TITLE
BreedingCostItemName

### ISSUE THREAD

**Issue by mczo13** [decaprime/LeadAHorseToWater#19]
Hello !  i m having trouble changing the item for breeding two horses. in the cfg i change the line for "BreedingCostItemName = Bone" for exemple, with bones on me , and it says that i need at least 1 in my inventory (i have 2500...) I tried "Wood" "Plant Fiber" "Plant Fibre" "Lumber" etc... nothing change. Any clue ?

Do you know what do i have to put in this line to breed with Plant Fiber for exemple?

Best regards

**Comment by thatWretch** [comment]
So I was struggling with this too, but what you're actually looking for is a value for BreedingRequiredItem (which is defaulted at -570287766). It's essentially the item ID, and for the life of me I couldn't find documentation online but maybe I'm blind. I do know you can check an item's GUID via the game's console commands though! With admin privileges, you can open the console with ` and type "give (item)" wherein item is the name, and the console with autofill to match - including what the GUID for that item is.

tldr; set "BreedingRequiredItem = -1409142667" for Plant Fibre
